tests/api/courses/GET.test.js

```diff
@@ -0,0 +1,77 @@
+const app = require("../../../app");
+const { createTables, wipeDBTables } = require("../../../db/databaseHelpers");
+const request = require("supertest");
+const createCourse = require("../../../helpers/Courses/createCourse");
+const createUser = require("../../../helpers/Users/createUser");
+const jwtGenerate = require("../../../helpers/jwtGenerate");
+
+let parentToken, studentToken;
+beforeAll(async() => {
+  await createTables();
+  await createUser("1", "student@email.com", "1234", "student");
+  await createUser("2", "tutor@email.com", "1234", "tutor");
+  await createUser("3", "parent@email.com", "1234", "parent");
+  parentToken = `user=${await jwtGenerate("3", "parent")}`;
+  studentToken = `user=${await jwtGenerate("3", "student")}`;
+});
+ 
+afterAll(async() =>{
+  await wipeDBTables();
+});
+
+describe("GET /api/courses", () =>{
+  describe("no token", () =>{
+    it("should return 401", async() =>{
+      const res = await request(app)
+        .get("/api/courses");
+      expect(res.status).toBe(401);
+    });
+  });
+
+  describe("token type is parent", () =>{
+    it("should return 401", async() =>{
+      const res = await request(app)
+        .get("/api/courses")
+        .set("Cookie", [parentToken]);
+      expect(res.status).toBe(401);
+    });
+  });
+  
+  describe("query string does not exist", () => {
```

Comment:
If there is no query string, just return all of the courses.

Reply:
c4f363c71f87c14edcda4ccab29f044dfc6d3963